tutorials/colors/colormapnorms.py

```diff
@@ -184,7 +185,44 @@
 
 pcm = ax[2].pcolormesh(X, Y, Z, cmap='RdBu_r', vmin=-np.max(Z))
 fig.colorbar(pcm, ax=ax[2], extend='both', orientation='vertical')
-fig.show()
+plt.show()
+
+
+###############################################################################
+# DivergingNorm: Different mapping on either side of a center
+# -----------------------------------------------------------
+#
+# Sometimes we want to have a different colormap on either side of a
+# conceptual center point, and we want those two colormaps to have
+# different linear scales.  An example is a topographic map where the land
+# and ocean have a center at zero, but land typically has a greater
+# elevation range than the water has depth range, and they are often
+# represented by a different colormap.
+
+filename = cbook.get_sample_data('exampleTopo.npz', asfileobj=False)
```

Comment:
Can you reuse the sample data from https://matplotlib.org/gallery/specialty_plots/topographic_hillshading.html instead? (and squash exampleTopo away)

Reply:
No because that has no water in it.  I don’t think it’s unrasonable to add a small geographic sample.  It’s not very big.

Reply:
45kb to be precise....